tests/src/dir.targets

```diff
@@ -66,7 +66,6 @@
   build a run only for a higher priority testcase.
   -->
   <Target Name="Build" Condition="('$(CLRTestPriority)' &lt;= '$(CLRTestPriorityToBuild)') Or ('$(_CLRTestInServiceOfHigherPriority)')">
-    <MSBuild Projects="@(ProjectReference)" Properties="_CLRTestInServiceOfHigherPriority=true"/>
```

Comment:
Would you suggest removing the other 3 references? And I'm actually not very familiar with MSbuild so I'm not sure - I was able to get a successful build locally with this change, and was going to use the CI's Pri1 test suite to see if those tests were getting built/run. Should I instead keep `<MSBuild Projects="@(ProjectReference)"` and remove `Properties="_CLRTestInServiceOfHigherPriority=true"` ?